data_integration/pipelines.py

```diff
@@ -150,8 +150,8 @@ def html_doc_items(self) -> [(str, str)]:
 
 class Pipeline(Node):
     nodes: {str: Node} = None
-    initial_node: Node = None
-    final_node: Node = None
+    initial_node: Task = None
+    final_node: Task = None
 
     def __init__(self, id: str,
```

Comment:
One can also add a pipeline or ParallelTask as initial / final node

Reply:
not really: there are places which expect a task, at least I had places where intelij complained that a method wasn't available

Reply:
Fixed it in a different way